### PR TITLE
AI content updates

### PR DESCRIPTION
**Pull request by danieltprice**
WIP Preview

Intro: https://neon-next-git-dprice-ai-content-updates-neondatabase.vercel.app/docs/ai/ai-intro
LangChain: https://neon-next-git-dprice-ai-content-updates-neondatabase.vercel.app/docs/ai/langchain
LlamaIndex: https://neon-next-git-dprice-ai-content-updates-neondatabase.vercel.app/docs/ai/llamaindex

### CODE REVIEW

**Comment by rishi-raj-jain** [comment]
Do we have this as WIP while we get all the documentation/examples as mentioned in the docs?

File: content/docs/ai/ai-intro.md [hunk 1]
```diff
@@ -7,18 +7,128 @@ updatedOn: '2023-10-07T10:43:33.361Z'
 
 Vector databases enable efficient storage and retrieval of vector data, which is an essential component in building AI applications that leverage Large Language Models (LLMs) such as OpenAI.
 
-Neon supports the `pgvector` open-source extension, which enables Postgres as a vector database for storing and querying vector embeddings. By enabling Postgres as a vector database, you can keep your data in the open source database that you know and trust. There's no need for data migration or a proprietary vector storage solution.
+Neon supports the `pgvector` open-source extension, which enables Postgres as a vector database for storing and querying vector embeddings. By enabling Postgres as a vector database, you can keep your data in the open-source database that you know and trust. There's no need for data migration or a proprietary vector storage solution.
```
Comment (rishi-raj-jain) [inline content/docs/ai/ai-intro.md:10]: proprietary -> 3rd party?

File: content/docs/ai/ai-intro.md [hunk 2]
```diff
@@ -7,18 +7,128 @@ updatedOn: '2023-10-07T10:43:33.361Z'
...
 
-To get started, see [The pgvector extension](/docs/extensions/pgvector).
+<CTA title="Neon AI Starter Kit" description="Get a headstart on your AI app with the Neon AI Starter Kit. Tools, starter apps, and step-by-step instructions to get you up and running in minutes." buttonText="Sign Up" buttonUrl="https://console.neon.tech/signup" />
 
-## Example applications
+The _Neon AI Starter Kit_ includes:
```
Comment (rishi-raj-jain) [inline content/docs/ai/ai-intro.md:14]: bold instead of italics?

File: content/docs/ai/ai-intro.md [hunk 3]
```diff
@@ -7,18 +7,128 @@ updatedOn: '2023-10-07T10:43:33.361Z'
...
 
-Check out the following AI application examples built with Neon.
+- Neon Postgres with the latest version of `pgvector` for storing vector embeddings
+- The `neon_vecs` Python and JavaScript SDKs for managing and querying vector embeddings in Postgres with the `pgvector` extension
+- The `neon_ai` utility for generating vector embeddings with your chosen LLM and storing them in Postgres
+- Starter apps to get you up and running in minutes:
+  - AI chatbot starter app
```
Comment (rishi-raj-jain) [inline content/docs/ai/ai-intro.md:20]: AI chatbot starter app - AI Chatbot starter application in Next.js

File: content/docs/ai/ai-intro.md [hunk 4]
```diff
@@ -7,18 +7,128 @@ updatedOn: '2023-10-07T10:43:33.361Z'
...
+  - RAG starter app
```
Comment (rishi-raj-jain) [inline content/docs/ai/ai-intro.md:21]: RAG starter app - AI RAG starter application in Next.js

File: content/docs/ai/ai-intro.md [hunk 5]
```diff
@@ -7,18 +7,128 @@ updatedOn: '2023-10-07T10:43:33.361Z'
...
+  - Semantic search starter app
```
Comment (rishi-raj-jain) [inline content/docs/ai/ai-intro.md:22]: Semantic search starter app - Semantic search starter application with Next.js and OpenAI

File: content/docs/ai/ai-intro.md [hunk 6]
```diff
@@ -7,18 +7,128 @@ updatedOn: '2023-10-07T10:43:33.361Z'
...
+  - Hybrid search starter app
```
Comment (rishi-raj-jain) [inline content/docs/ai/ai-intro.md:23]: Hybrid search starter app - Hybrid search starter application with Next.js and OpenAI

File: content/docs/ai/ai-intro.md [hunk 7]
```diff
@@ -7,18 +7,128 @@ updatedOn: '2023-10-07T10:43:33.361Z'
...
+  - Image search starter app
```
Comment (rishi-raj-jain) [inline content/docs/ai/ai-intro.md:24]: Image search starter app - Image similarity search starter application with Next.js and OpenAI

File: content/docs/ai/ai-intro.md [hunk 8]
```diff
@@ -7,18 +7,129 @@ updatedOn: '2023-10-07T10:43:33.361Z'
...
+  - Chat with PDF starter app
+- The Neon AI scaling guide
```
Comment (rishi-raj-jain) [inline content/docs/ai/ai-intro.md:26]: can we expand on this in here - helps users like me know what to expect if I click on it.

File: content/docs/ai/ai-intro.md [hunk 9]
```diff
@@ -7,18 +7,129 @@ updatedOn: '2023-10-07T10:43:33.361Z'
...
+- The Neon AI go-live checklist
```
Comment (rishi-raj-jain) [inline content/docs/ai/ai-intro.md:27]: I think I can infer - but can we still expand in a single line as to what this is?

File: content/docs/ai/ai-intro.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/ai/ai-intro.md:10]: That's probably more relatable language. Changed!

File: content/docs/ai/ai-intro.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/ai/ai-intro.md:14]: changed!

File: content/docs/ai/ai-intro.md [hunk 10]
```diff
@@ -7,18 +7,152 @@ updatedOn: '2023-10-07T10:43:33.361Z'
...
+- Hackable, fully-featured [starter apps](#ai-starter-apps) to get you up and running:
+  - Chatbot starter apps
+  - RAG chatbot starter apps
+  - Semantic search starter apps
+  - Hybrid search starter app
+  - Reverse image search starter apps
+  - Chat with PDF starter apps
+- The [Neon AI scaling guide](/docs/ai/ai-scale-with-neon) &#8212; scale your app with Neon's Autoscaling, Read Replica, and serverless driver features
+- The Neon AI go-live checklist &#8212; make sure you are ready for production with our go-live checklist
+- A collection of [AI apps built with Neon](#ai-apps-built-with-neon) for your reference
```
Comment (rishi-raj-jain) [inline content/docs/ai/ai-intro.md:26]: "for your reference" => "to get started quickly"?

File: content/docs/ai/ai-intro.md [hunk 11]
```diff
@@ -7,18 +7,119 @@ updatedOn: '2023-10-07T10:43:33.361Z'
...
+
+<a href="ai-vector-search-optimization" description="Optimize vector search for better application performance" icon="openai">Vector search optimization</a>
+
+<a href="/docs/ai/ai-scale-with-neon" description="Scale your app with Neon's Autoscaling, Read Replica, and serverless driver features" icon="openai">Neon AI scaling guide</a>
+
+</DetailIconCards>
+
+## AI starter apps
+
+Hackable, fully-featured, pre-built [starter apps](#ai-starter-apps) to get you up and running.
```
Comment (rishi-raj-jain) [inline content/docs/ai/ai-intro.md:51]: presently, this hyperlink links it to the same section it's in.

how about we update it to point to https://github.com/neondatabase/examples/tree/main/ai

File: content/docs/ai/ai-intro.md [hunk 12]
```diff
@@ -7,18 +7,119 @@ updatedOn: '2023-10-07T10:43:33.361Z'
...
 
 <DetailIconCards>
-<a href="https://github.com/neondatabase/yc-idea-matcher" description="Build an AI-powered semantic search application" icon="github">Semantic search app</a>
-<a href="https://github.com/neondatabase/ask-neon" description="Build an AI-powered chatbot with pgvector" icon="github">Chatbot app</a>
-<a href="https://vercel.com/templates/next.js/postgres-pgvector" description="Enable vector similarity search with Vercel Postgres" icon="github">Vercel Postgres pgvector Starter</a>
-<a href="https://github.com/neondatabase/postgres-ai-playground" description="Build an AI-enabled SQL playground for natural language queries" icon="github">Web-based AI SQL Playground</a>
+
+<a href="https://github.com/neondatabase/examples/tree/main/ai/llamaindex/chatbot-nextjs" description="A fully-featured, hackable Netx.js AI chatbot built with OpenAI and LlamaIndex" icon="github">AI chatbot (OpenAI + LllamIndex)</a>
```
Comment (rishi-raj-jain) [inline content/docs/ai/ai-intro.md:55]: Netx.js ==> Next.js

File: content/docs/ai/ai-intro.md [hunk 13]
```diff
@@ -7,18 +7,119 @@ updatedOn: '2023-10-07T10:43:33.361Z'
...
+
+<a href="https://github.com/neondatabase/examples/tree/main/ai/langchain/chatbot-nextjs" description="A fully-featured, hackable Netx.js AI chatbot built with OpenAI and LangChain" icon="github">AI chatbot (OpenAI + LangChain)</a>
```
Comment (rishi-raj-jain) [inline content/docs/ai/ai-intro.md:57]: Netx.js ==> Next.js

File: content/docs/ai/ai-intro.md [hunk 14]
```diff
@@ -7,18 +7,119 @@ updatedOn: '2023-10-07T10:43:33.361Z'
...
+
+<a href="https://github.com/neondatabase/examples/tree/main/ai/langchain/rag-nextjs" description="A fully-featured, hackable Next.js RAG chatbot built with OpenAI and LangChain" icon="github">RAG chatbot (OpenAI + LangChain)</a>
+
+<a href="https://github.com/neondatabase/examples/tree/main/ai/llamaindex/semantic-search-nextjs" description="A fully-featured, hackable Next.js Semantic Search chatbot built with OpenAI and LlamaIndex" icon="github">Semantic search chatbot (OpenAI + LlamaIndex)</a>
+
+<a href="https://github.com/neondatabase/examples/tree/main/ai/langchain/semantic-search-nextjs" description="A fully-featured, hackable Next.js Semantic Search chatbot built with OpenAI and LangChain" icon="github">Semantic search chatbot (OpenAI + LangChain)</a>
+
+<a href="https://github.com/neondatabase/examples/tree/main/ai/hybrid-search-nextjs" description="A full-featured, hackable Next.js Hybrid Search built with OpenAI" icon="github">Hybrid search (OpenAI)</a>
```
Comment (rishi-raj-jain) [inline content/docs/ai/ai-intro.md:67]: can we move this to the top?

**Review by rishi-raj-jain — APPROVED** [review]

**Pull request merged**